src/Utils/BlockTemplateUtils.php

```diff
@@ -422,17 +422,21 @@ public static function theme_has_template_part( $template_name ) {
 	/**
 	 * Checks to see if they are using a compatible version of WP, or if not they have a compatible version of the Gutenberg plugin installed.
 	 *
+	 * @param  boolean $template_part Whether we are checking for a template part or not.
 	 * @return boolean
 	 */
-	public static function supports_block_templates() {
-		if (
-			! wc_current_theme_is_fse_theme() &&
-			( ! function_exists( 'gutenberg_supports_block_templates' ) || ! gutenberg_supports_block_templates() )
-		) {
-			return false;
+	public static function supports_block_templates( $template_part = false ) {
```

Comment:
I'll have to doublecheck this change. My memory is foggy, I'm not sure if this will potentially introduce warnings for > PHP 8 because of the change in signature. However, given this provides a default value it should be okay.